charts/fybrik/files/taxonomy/module.values.schema.json

```diff
@@ -1,100 +0,0 @@
-{
```

Comment:
This file is in use.

Reply:
Removed the test that was referencing this old schema